### DISCUSSION TITLE
Why "alpha-nvim"  / "dashboard-nvim" are always disabled when restart nvim ?

### DISCUSSION THREAD

**Discussion by Cebastion** [LazyVim/LazyVim#4778] [Q&A] [answered]
![Screenshot from 2024-11-12 19-56-23](https://github.com/user-attachments/assets/e37aa786-9d80-4f7b-a359-1d109fe48211)
![Screenshot from 2024-11-12 19-56-06](https://github.com/user-attachments/assets/4da7aa74-8730-48e8-afe5-d331187a83d3)
![Screenshot from 2024-11-12 19-55-40](https://github.com/user-attachments/assets/86285db4-fa1a-4de4-ad7a-eb2365bccd8b)

**Comment by dpetka2001** [comment]
The only time this happens is when you've enabled `mini.starter` to use as your dashboard.

**Comment by Cebastion** [comment]
![Screenshot from 2024-11-12 20-17-24](https://github.com/user-attachments/assets/9ce581c8-0fb2-404b-aa86-4fc2d9a64dc1)
![Screenshot from 2024-11-12 20-17-16](https://github.com/user-attachments/assets/a93c8e4d-16d8-4981-b371-e7ccc0dad16c)
![Screenshot from 2024-11-12 20-16-54](https://github.com/user-attachments/assets/2293daa1-1562-4064-baab-e7cd59c56c83)

So mini.started is active, but it doesn't work. I deleted file dashboard and alpha
@dpetka2001

**Reply by dpetka2001** [reply]
You can only have one dashboard active. Choose which one of the 3 you want. I only use default dashboard and am happy with it.

**Reply by Cebastion** [reply]
How? I turn off dashboard and alpha, but it doesn't work :(

**Reply by dpetka2001** [reply, marked as answer]
You have probably enabled `mini.starter` from the LazyExtras. Just disable it if you don't want it.

**Reply by Cebastion** [reply]
So, I fixed it. Need  add { import = "lazyvim.plugins.extras.ui.mini-starter" } in file lazy.lua. Anyway, thank you for help

**Reply by dpetka2001** [reply]
Don't do it that way. You better use LazyExtras to enable the Extras you want instead of importing them into your `lazy.lua` file. You might run into problems if you don't know what you're doing.
